samples/nrf9160/lwm2m_client/sample_description.rst

```diff
@@ -523,6 +523,7 @@ The following files are available:
 * :file:`overlay-assist-cell.conf` - Enables cell-based location assistance.
 * :file:`overlay-assist-pgps.conf` - Enables P-GPS assistance in the sample.
 * :file:`overlay-lowpower.conf` - Disables certain features to bring the power consumption down.
+* :file:`overlay-adv-firmware.conf` - Enable multi instance firmware update object. Modem and App have own instances.
```

Comment:
```suggestion
* :file:`overlay-adv-firmware.conf` - Enable experimental Advanced Firmware Update object.
```

Reply:
Just adding one more minor change on top of Seppo's suggestion.
```suggestion
* :file:`overlay-adv-firmware.conf` - Enables experimental Advanced Firmware Update object.
```